lib/xcode/install.rb

```diff
@@ -48,6 +48,18 @@ def current_symlink
       File.symlink?(SYMLINK_PATH) ? SYMLINK_PATH : nil
     end
 
+    def os_version_compatibility_issue?(version)
```

Comment:
I will add a few comments describing the issue with<10.11.5 failing to extract the archive